Jenkinsfile

```diff
@@ -6,9 +6,7 @@ def master_branches = ["master", ] as String[]
 
 pipeline {
   agent {
-    dockerfile {
-      args  "--shm-size=1g"
-    }
+    label 'dcos-ui'
```

Comment:
can you explain why we would want to use the label instead of our dockerfile? 🤔

Reply:
to avoid docker in docker and gain some speed

Reply:
following recommendations of Patric